ibtk/src/lagrangian/LData.cpp

```diff
@@ -139,36 +119,14 @@ LData::LData(Pointer<Database> db) : d_name(db->getString("d_name")), d_depth(db
     if (num_ghost_nodes > 0)
     {
         db->getIntegerArray("d_nonlocal_petsc_indices",
-                            d_nonlocal_petsc_indices.empty() ? nullptr : &d_nonlocal_petsc_indices[0],
+                            d_nonlocal_petsc_indices.empty() ? nullptr : d_nonlocal_petsc_indices.data(),
```

Comment:
Do we still need this check with `data()`? I think we can just do:
```suggestion
                            d_nonlocal_petsc_indices.data(),
```